python/GafferSceneUI/LightEditor.py

```diff
@@ -549,6 +590,14 @@ def __buttonPress( self, pathListing, event ) :
 					"shortCut" : "D",
 				}
 			)
+			menuDefinition.append(
+				"Remove Attribute",
```

Comment:
We seem to have inconsistent terminology between the DeleteAttributes node and the AttributeTweaks node and its  Remove mode. I think my preference would be to standardise on "Delete", but I can't really articulate why. Any thoughts?

Reply:
Just posting here that we decided offline to not worry about changing this, at least partly because it might chain react into changing the `Remove` tweak mode enum.